### DISCUSSION TITLE
Can we have a weekly or bi-weekly release cycle?

### DISCUSSION THREAD

**Discussion by Ashish0804** [blackjack4494/youtube-dlc#168] [General]
Finally yt-dl(c) is receiving constant fixes, so it will be very helpful if we can have a weekly release cycle.

**Why not just run from the source if u need a fix fast?**

Because yt-dl(c) doesn't work well on some platforms when running from source, e.g.; colab.

**Comment by blackjack4494** [comment]
There is no fixed schedule for releases. As of now it's rather if there is enough or big changes for a new release there will be a new release. Tho I still had some weekly-ish cycle in my mind.  
We can elaborate on this as I have no strict schedule as stated above.
